feature-libs/organization/user-registration/components/form/user-registration-form.component.html

```diff
@@ -0,0 +1,287 @@
+<section class="container" *ngIf="!(isLoading$ | async); else loading">
+  <div role="status" [attr.aria-label]="'common.loaded' | cxTranslate"></div>
+  <div class="row justify-content-center">
+    <div class="col-md-6">
+      <div class="cx-section">
```

Comment:
are those all `div`s necessary? 
weren't we supposed to deviate from the bootstrap classes and do styling by the selectors?